.changeset/perfect-scissors-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@inlang/paraglide-js": patch
+---
+
+feat: Add options argument to messageIndex function, to allow forcing a locale
```

Comment:
Can you write the changelog manually? 

Changeset doesn't work with paraglide at the moment because of the pre-release stuff. Besides, this is a minor bumb not a patch under SemVer. Good news, we decided to get rid of patches altogether and just have `MAJOR-MINOR` where `MAJOR` is a breaking change/needs user attention. MINOR is an update/new feature/fix/whatever that does not require user attention.

Reply:
Ok. I removed the changeset and added a section to the changelog myself. Does this mean I also have to bump the package version manually?

I also added a test in `compile.test.ts`

Reply:
@LorisSigrist yes. Only I can publish atm. If you write the release notes and merge, I can bump the version and publish manually.